packages/sync-repo-settings/package.json

```diff
@@ -12,13 +12,14 @@
   "files": [
     "build/src"
   ],
+  "bin": "build/src/bin/sync-repo-settings.js",
   "keywords": [
     "probot",
     "github",
     "probot-app"
   ],
   "scripts": {
-    "compile": "tsc",
+    "compile": "tsc -p .",
```

Comment:
So technically you shouldn't need that!  The TypeScript compiler defaults to checking for a `./tsconfig.json`.